src/sentry/south_migrations/0455_auto__add_field_groupenvironment_first_seen.py

```diff
@@ -0,0 +1,1245 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
```

Comment:
Let's make this True, and just coordinate apply this offline with @getsentry/ops since this is a really busy table.

Reply:
This is also not ideal because it's also adding an index, which will be blocking too.

So we need raw SQL to apply both the new column, and creating the index concurrently.